python/services/slack_service.py

```diff
@@ -0,0 +1,26 @@
+from slack_sdk import WebClient
+
+from python.services.sentry_service import UsageStats
+
+
+class SlackService:
+    OPERATIONS_ENGINEERING_ALERTS_CHANNEL_ID = "C033QBE511V"
+
+    # Added to stop TypeError on instantiation. See https://github.com/python/cpython/blob/d2340ef25721b6a72d45d4508c672c4be38c67d3/Objects/typeobject.c#L4444
+    def __new__(cls, *_, **__):
+        return super(SlackService, cls).__new__(cls)
+
+    def __init__(self, slack_token: str) -> None:
+        self.slack_client = WebClient(slack_token)
+
+    def send_error_usage_alert_to_operations_engineering(self, period_in_days: int, usage_stats: UsageStats,
+                                                         usage_threshold: float):
+        self.slack_client.chat_postMessage(channel=self.OPERATIONS_ENGINEERING_ALERTS_CHANNEL_ID,
+                                           mrkdown=True,
+                                           text=f"*Sentry Errors have exceeded {usage_threshold:.2%} usage in the past {period_in_days} days*\n`This is a test message for Sentry Error alerts!` :test_tube:\nError quota consumed over past {period_in_days} days [ {usage_stats.total} / {usage_stats.max_usage} ]\nPercentage consumed: [ {usage_stats.percentage_of_quota_used:.2%} ]")
```

Comment:
👆 messages have some test message text for the moment, just in case I forget to stick it in the channel 😂